api/app/Models/User.php

```diff
@@ -96,8 +98,90 @@ class User extends Model implements Authenticatable, LaratrustUser
     protected $fillable = [
         'email',
         'sub',
+        'searchable',
     ];
 
+    protected $hidden = [
+        'searchable',
+    ];
+
+    /**
+     * Get the indexable data array for the model.
+     *
+     * @return array<string, mixed>
+     */
+    public function toSearchableArray(): array
+    {
+        $this->loadMissing([
+            'poolCandidates',
+            'workExperiences',
+            'educationExperiences',
+            'personalExperiences',
+            'communityExperiences',
+            'awardExperiences',
+        ]);
+
+        $notes = $this->poolCandidates->pluck('notes')->toArray();
```

Comment:
Hmm, this might be a bit awkward.  If you're indexing all the pool candidate notes and you use this index to search pool canidates you might get hits for the notes on pool candidates that you're not able to see.